Cython/Includes/cpython/exc.pxd

```diff
@@ -88,6 +88,11 @@ cdef extern from "Python.h":
     # needs to handle exceptions or by code that needs to save and
     # restore the error indicator temporarily.
 
+    PyObject* PyErr_GetHandledException()
+    void PyErr_SetHandledException(PyObject* exc)
+    PyObject* PyErr_GetRaisedException()
+    void PyErr_SetRaisedException(PyObject* exc)
```

Comment:
I checked the documentation and the implementation. For the last two functions, the documentation doesn't say it, but the implementation allows `NULL` to be returned and passed, so `PyObject*` is correct here (and not `object`).
This is also the case for the first two functions that you added above.